src/scripts/OutSystems/OSUI/Utils/Utilities.ts

```diff
@@ -26,6 +26,7 @@ namespace OutSystems.OSUI.Utils {
 	 */
 	// eslint-disable-next-line @typescript-eslint/naming-convention
 	export function GetIsRTL(): boolean {
+		// This method can't implement the CreateApiResponse method since it's defined as a function at SS
```

Comment:
Should be:
"This method can't implement the CreateApiResponse method since it's defined as a function in SS"